client/src/components/DataFiles/DataFilesModals/DataFilesManageProjectModal.jsx

```diff
@@ -137,7 +137,7 @@ const DataFilesManageProjectModal = () => {
           ) : null}
           <div className={styles['owner-controls']}>
             {isOwner && members.length > 1 ? (
-              <Button color="link" onClick={toggleTransferMode}>
+              <Button type="link" onClick={toggleTransferMode}>
                 <h6 className={styles['ownership-toggle']}>
```

Comment:
you dropped ownership-toggle dropped in `DataFilesManageProject.module.scss` 👍 but looks like it can be dropped here.